shogun-lib/src/main/java/de/terrestris/shogun/lib/service/security/provider/GroupProviderService.java

```diff
@@ -33,4 +33,5 @@ public interface GroupProviderService<UserType, GroupType> {
 
     Group<GroupType> findOrCreateByProviderId(String providerGroupId);
 
+    void createAllGroups();
```

Comment:
Just wondering, do we need to flag this commit as breaking?

Reply:
It's not breaking anything, is it? It's a new feature.

Reply:
Since it's changing an interface, project implementations would need to include the implementation of the newly added method.